src/kernel/platform.h

```diff
@@ -32,7 +33,10 @@ namespace rt {
  */
 class Platform {
 public:
-    Platform() {}
+    Platform() {
+      time_ = CMOSTime();
```

Comment:
this is actually not necessary, because `CMOSTime` is not a POD type and will be initialized using default constructor.
Nice SO explanation:
http://stackoverflow.com/questions/15212261/default-initialization-of-pod-types-in-c